nginx/frontend

```diff
@@ -4,6 +4,11 @@ server {
     location ~ ^(?:/[-a-z/]+)?/static/(.+)$ {
         alias /app/app/static/$1;
 
+        gzip on;
+        gzip_disable "msie6";
```

Comment:
I don't think it's worth adding this.
I've just opened IE6 in browserstack and DMp does even load:
![image](https://user-images.githubusercontent.com/3441519/67476784-9ccce100-f650-11e9-9f63-5f9dbe5dc9f0.png)

Reply:
Cool thanks for checking - I just copied this configuration from the router, and it hadn't been touched in a long time there.

Reply:
Wow old skool